agent/config/pbench-agent.cfg.example

```diff
@@ -40,8 +40,16 @@ interval = 30
 # pandas-package = python-pandas
 # Fedora has python2-pandas and python3-pandas
 # pandas-package = python2-pandas
+stockpile_dependencies = git, ansible
```

Comment:
Why does stockpile depend on `git`?

Reply:
We don't have stockpile packaged, so we are cloning the repo for now. Eventually we will be packaging and installing a stable version of stockpile.

Reply:
We have `check_install_rpm` in the `agent/base` include which should take care of such checks.

But let's change this to assume the stockpile code is "part" of the `pbench-agent` install using `git submodules` [(1)](https://git-scm.com/docs/git-submodule).

Further, it seems best to make the `pbench-agent` RPM depend on ansible now, as we are using it in other places in the code, too.

Reply:
Agree with making the pbench-agent RPM depend on ansible. Will take a look at git submodules  @portante.

Reply:
So now that we are using git sub modules, do we still need this?